digdag-server/src/main/java/io/digdag/server/rs/UiResource.java

```diff
@@ -57,6 +65,21 @@ public Response getApplication(@PathParam("path") String path)
             .or(indexResponse);
     }
 
+    @GET
+    @Path("/config.js")
+    public Response getConfigJs()
+            throws IOException
+    {
+        final Optional<String> uiConfigJsPath = serverConfig.getUiConfigJsPath();
+        if (uiConfigJsPath.isPresent()) {
+            final Optional<Response> response = getResourceIfExists(uiConfigJsPath.get());
```

Comment:
`getResourceIfExists` can't get the resource of the local file. 
Always I saw an exception (`java.lang.IllegalArgumentException: resource $path/config.js not found.`) although the config file exists.

So how about this?

```suggestion
            final Optional<Response> response = getLocalResourceIfExists(uiConfigJsPath.get());
```

The details are as follows:

```diff
diff --git digdag-server/src/main/java/io/digdag/server/rs/UiResource.java digdag-server/src/main/java/io/digdag/server/rs/UiResource.java
index 27063c0f9..35028e2dc 100644
--- digdag-server/src/main/java/io/digdag/server/rs/UiResource.java
+++ digdag-server/src/main/java/io/digdag/server/rs/UiResource.java
@@ -1,10 +1,15 @@
 package io.digdag.server.rs;

+import com.google.common.base.Charsets;
 import com.google.common.base.Optional;
+import com.google.common.io.Files;
 import com.google.common.io.Resources;
 import com.google.inject.Inject;
 import io.digdag.server.ServerConfig;

+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.net.URI;
 import java.net.URL;
@@ -72,7 +77,7 @@ public class UiResource
     {
         final Optional<String> uiConfigJsPath = serverConfig.getUiConfigJsPath();
         if (uiConfigJsPath.isPresent()) {
-            final Optional<Response> response = getResourceIfExists(uiConfigJsPath.get());
+            final Optional<Response> response = getLocalResourceIfExists(URI.create(uiConfigJsPath.get()).getPath());
             if (response.isPresent()) {
                 return response.get();
             }
@@ -100,4 +105,16 @@ public class UiResource
             return Optional.absent();
         }
     }
+
+    private Optional<Response> getLocalResourceIfExists(String name)
+    {
+        try {
+            BufferedReader br = Files.newReader(new File(name), Charsets.UTF_8);
+            Response response = Response.ok(br).type(URLConnection.guessContentTypeFromName(name)).build();
+            return Optional.of(response);
+        }
+        catch (FileNotFoundException e) {
+            return Optional.absent();
+        }
+    }
 }
```